### ISSUE TITLE
[Bug]: [bulkload] failed to bulk load data if multiple data nodes

### ISSUE THREAD

**Issue by yanliang567** [milvus-io/milvus#17300]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Environment

```markdown
- Milvus version: master-20220531-d920e5c9
- Deployment mode(standalone or cluster): cluster with multiple data nodes
- SDK version(e.g. pymilvus v2.0.0rc2): pymilvus 2.1.0.dev67
```

### Current Behavior

bulk load tasks completed with persisted state, but the data was not available.
it first hang at collection.num_entities(), and then all the following bulk load tasks failed with timeout.

### Expected Behavior

bulk load data successfully and data available for search and query.

### Steps To Reproduce

```markdown
1. deploy milvus cluster with multiple datanodes
2. bulk load a json file
```

### Milvus Log

[bulk.fail.log.tar.gz](https://github.com/milvus-io/milvus/files/8804730/bulk.fail.log.tar.gz)

**Comment by yhmo** [comment]
Fixed by #17324

**Comment by yanliang567** [comment]
verified fix on master-20220602-48cf63a2
